templates/zerver/back_to_login_component.html

```diff
@@ -0,0 +1,3 @@
+<div class="actions if-zulip-electron"><!-- only show if on `ZulipElectron` -->
+    <a class="back-to-login" href="{{login_url}}"><i class="fa fa-arrow-left"></i> Back to the login page</a>
+</div>
```

Comment:
Let's put this under `templates/zerver/include/`, just to make clear it's a fragment.